docs/scaling/downscaling.txt

```diff
@@ -0,0 +1,231 @@
+.. _scaling-down:
+
+===========
+Downscaling
+===========
+
+In this guide we:
+
+- Create a Vanilla cluster.
+- Add some data to it.
+- Downscale it to a single node cluster.
+
+
+.. _scaling-down-starting-vanilla-cluster:
+
+Starting a Vanilla cluster
```

Comment:
Is "Vanilla" a technical term here, or just meant in the sense of "generic"? If the latter I'd write it lowercase "vanilla"